examples/frameworks/adk_demo/pyproject.toml

```
@@ -0,0 +1,24 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools >= 64", "setuptools-scm>=8"]
+
+[tool.setuptools_scm]
+root = "../../.."
+
+[project]
+name = "nat_adk_demo"
+dynamic = ["version"]
+dependencies = [
+  "nvidia-nat[adk]~=1.2",
+  "matplotlib~=3.9" 
+]
+requires-python = ">=3.11,<3.13"
```

We just added python 3.13 support!
```suggestion
requires-python = ">=3.11,<3.14"
```